File: example06/visualstudio/dllExample06/dllExample06/Information.cpp
#include "pch.h"
#include "Handle.hpp"
#include "Information.hpp"

Information::Information()
{
	info = NULL;
}

Information::~Information()
{

}

void Information::setInformation(const char* msg)
{
	info = msg;
}

const char* Information::getInformation()
{
	return info;
}

File: example06/eclipse/testExample06/src/example06C/Information.java
package example06C;

public class Information extends NativePointer
{
	// this constructor will also call "NativePointer()" which natively initializes the pointer
	public Information() throws NativeCreationException {} 
	
	public Information(long pointer) 
	{
		super(pointer);
	}
	
	public native void setInformation(String msg);
	
	public native String getInformation();
}
